package.json

```diff
@@ -34,6 +34,10 @@
   ],
   "typings": "lib/index.d.ts",
   "license": "MIT",
+  "peerDependencies": {
+    "react": ">=15.0.0",
+    "react-dom": ">=15.0.0"
```

Comment:
I think we should set version to `*`.

Reply:
Are older react versions supported?

Reply:
I'm not sure, but antd doesn't require a particular version of react currently.

Reply:
Setting it to `>=15.0.0` should cover all modern react versions. I picked it up from the `devDependecies` of `antd`'s package.json itself. I'm guessing all of the tests for `antd` assume `react>=15.0.0`. If you feel otherwise, we could downgrade.

Reply:
I tested antd against React 0.14, all tests can pass after https://github.com/react-component/calendar/pull/298 merged.

Reply:
We can set it to `"~0.14.0 || >=15.0.0"`

Reply:
@pastelsky can you set it to "~0.14.0 || >=15.0.0" and repull